optimade/server/models.py

```diff
@@ -53,24 +58,110 @@ class StructureResource(EntryResource):
     attributes: StructureResourceAttributes
 
 
-class OptimadeResponseMeta(BaseModel):
+class OptimadeProvider(BaseModel):
+    """ Stores information on the database provider of the
+    implementation.
+
     """
-    A JSON API meta member that contains JSON API meta objects of non-standard meta-information.
 
-    In addition to the required fields, it MAY contain
+    name: str = Schema(
+        ...,
+        description="a short name for the database provider"
+    )
+
+    description: str = Schema(
+        ...,
+        description="a longer description of the database provider"
+    )
 
-    - `data_available`: an integer containing the total number of data objects available in the database.
-    - `last_id`: a string containing the last ID returned.
-    - `response_message`: response string from the server.
+    prefix: str = Schema(
+        ...,
+        description="database-provider-specific prefix as found in "
+                    "Appendix 1."
+    )
 
-    Other OPTIONAL additional information global to the query that is not specified in this document, MUST start with
-    a database-provider-specific prefix
+    homepage: Optional[Union[UrlStr, Link]] = Schema(
+        ...,
+        description="a [JSON API links object](http://jsonapi.org/format/#document-links) "
+                    "pointing to homepage of the database provider, either "
+                    "directly as a string, or as a link object."
```

Comment:
looking at the json api spec it's not so clear what a "link object" is... is that something we define?

Reply:
This is hopefully something that #12 will make clear...